<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>XrayImagingWidgetVerticalLayout</class>
 <widget class="QWidget" name="XrayImagingWidgetVerticalLayout">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>784</width>
    <height>179</height>
   </rect>
  </property>
  <property name="sizePolicy">
   <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
    <horstretch>0</horstretch>
    <verstretch>0</verstretch>
   </sizepolicy>
  </property>
  <property name="maximumSize">
   <size>
    <width>32767</width>
    <height>32767</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>Acquisition widget</string>
  </property>
  <layout class="QGridLayout" name="gridLayout">
   <item row="0" column="0">
    <layout class="QGridLayout" name="gridLayout_2">
     <item row="1" column="1">
      <widget class="QLineEdit" name="ff_path_ledit"/>
     </item>
     <item row="1" column="2">
      <widget class="QPushButton" name="ff_browse_button">
       <property name="text">
        <string>Browse</string>
       </property>
      </widget>
     </item>
     <item row="0" column="1">
      <widget class="QLineEdit" name="data_path_ledit"/>
     </item>
     <item row="0" column="0">
      <widget class="QLabel" name="data_path_label">
       <property name="text">
        <string>Data:</string>
       </property>
      </widget>
     </item>
     <item row="0" column="2">
      <widget class="QPushButton" name="data_browse_button">
       <property name="text">
        <string>Browse</string>
       </property>
      </widget>
     </item>
     <item row="1" column="0">
      <widget class="QLabel" name="ff_path_label">
       <property name="text">
        <string>Flat field: </string>
       </property>
      </widget>
     </item>
     <item row="2" column="0" colspan="2">
      <widget class="QCheckBox" name="ff_apply_cbox">
       <property name="text">
        <string>Apply flat field correction</string>
       </property>
      </widget>
     </item>
     <item row="2" column="2">
      <widget class="QPushButton" name="load_images_button">
       <property name="text">
        <string>Load</string>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item row="1" column="0">
    <layout class="QHBoxLayout" name="horizontalLayout">
     <item>
      <widget class="QPushButton" name="first_image_button">
       <property name="minimumSize">
        <size>
         <width>27</width>
         <height>27</height>
        </size>
       </property>
       <property name="maximumSize">
        <size>
         <width>27</width>
         <height>27</height>
        </size>
       </property>
       <property name="text">
        <string/>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="prev_image_button">
       <property name="minimumSize">
        <size>
         <width>27</width>
         <height>27</height>
        </size>
       </property>
       <property name="maximumSize">
        <size>
         <width>27</width>
         <height>27</height>
        </size>
       </property>
       <property name="text">
        <string/>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="minus_quarter_button">
       <property name="maximumSize">
        <size>
         <width>27</width>
         <height>16777215</height>
        </size>
       </property>
       <property name="text">
        <string>-90</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QSlider" name="image_slider">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="plus_quarter_button">
       <property name="maximumSize">
        <size>
         <width>27</width>
         <height>16777215</height>
        </size>
       </property>
       <property name="text">
        <string>+90</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="next_image_button">
       <property name="maximumSize">
        <size>
         <width>27</width>
         <height>27</height>
        </size>
       </property>
       <property name="text">
        <string/>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="last_image_button">
       <property name="maximumSize">
        <size>
         <width>27</width>
         <height>27</height>
        </size>
       </property>
       <property name="text">
        <string/>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QSpinBox" name="image_spinbox"/>
     </item>
     <item>
      <widget class="QPushButton" name="play_button">
       <property name="minimumSize">
        <size>
         <width>27</width>
         <height>27</height>
        </size>
       </property>
       <property name="maximumSize">
        <size>
         <width>27</width>
         <height>27</height>
        </size>
       </property>
       <property name="text">
        <string/>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="stop_button">
       <property name="minimumSize">
        <size>
         <width>27</width>
         <height>27</height>
        </size>
       </property>
       <property name="maximumSize">
        <size>
         <width>27</width>
         <height>27</height>
        </size>
       </property>
       <property name="text">
        <string/>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QCheckBox" name="repeat_cbox">
       <property name="text">
        <string>Repeat</string>
       </property>
       <property name="checked">
        <bool>true</bool>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item row="0" column="1">
    <spacer name="horizontalSpacer">
     <property name="orientation">
      <enum>Qt::Horizontal</enum>
     </property>
     <property name="sizeHint" stdset="0">
      <size>
       <width>40</width>
       <height>20</height>
      </size>
     </property>
    </spacer>
   </item>
  </layout>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <pixmapfunction>qPixmapFromMimeSource</pixmapfunction>
 <resources/>
 <connections/>
</ui>
